test/bin/scenario.sh

```
@@ -1224,6 +1224,26 @@ USHIFT_USER: "${USHIFT_USER:-redhat}"
+            local other_vm
+            other_vm=$(basename "${vm_dir}")
+            [ "${other_vm}" = "${vmname}" ] && continue
+
+            local var_prefix
+            var_prefix=$(echo "${other_vm}" | tr '[:lower:]-' '[:upper:]_')
+            for prop in ip ssh_port api_port lb_port; do
+                local prop_file="${vm_dir}/${prop}"
+                [ -f "${prop_file}" ] || continue
+                local val
+                val=$(cat "${prop_file}")
+                local var_name="${var_prefix}_$(echo "${prop}" | tr '[:lower:]' '[:upper:]')"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)